runtime/src/main/scala/akka/grpc/GrpcProtocol.scala

```diff
@@ -88,6 +88,8 @@ object GrpcProtocol {
       messageEncoding: Codec,
       /** Encodes a frame as a part in a chunk stream. */
       encodeFrame: Frame => ChunkStreamPart,
+      /** A shortcut to encode a data frame directly into a ByteString */
+      encodeDataToFrameBytes: ByteString => ByteString,
```

Comment:
It's a bit unclear for me which parts of the runtime are really internal, and which are used from the generated code and subject to the binary compatibility guarantees.

Reply:
yeah, that's tricky - we try to use `@InternalStableApi` for those places we know are used from the generated code, but it's hard to guarantee that is exhaustive. Adding this field looks fine to me, though!